src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/__init__.py

```diff
@@ -12,8 +12,8 @@ class KeyVaultCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        from azure.cli.command_modules.keyvault._client_factory import keyvault_client_factory
-        from azure.cli.command_modules.keyvault._command_type import KeyVaultCommandGroup, KeyVaultArgumentContext
+        from ._client_factory import keyvault_client_factory
+        from ._command_type import KeyVaultCommandGroup, KeyVaultArgumentContext
         keyvault_custom = CliCommandType(
             operations_tmpl='azure.cli.command_modules.keyvault.custom#{}',
             client_factory=keyvault_client_factory
```

Comment:
Is this going to supercede the keyvault extension?  If so, you need to add annotations here to indicate that. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#extension-suppression